tests/Api/CategoryApiIntegration.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+namespace Akeneo\Pim\tests\Api;
+
+
+use Akeneo\Pim\Api\CategoryApi;
+
+class CategoryApiIntegration extends ApiTestCase
```

Comment:
Maybe better to split it as for the PIM: one class for each method (you can put listPerPage and all in the same file). We will probably enrich the tests over the time (new tests or new methods/endpoints).